drivers/ethernet/eth_mcux.c

```diff
@@ -39,7 +39,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #include "fsl_enet.h"
 #include "fsl_phy.h"
-#ifdef CONFIG_NET_POWER_MANAGEMENT
+#if defined(CONFIG_NET_POWER_MANAGEMENT)
```

Comment:
Is this type of change just preference for consistency or do we have a standard to use '#if defined()' instead of '#ifdef'?

Reply:
In the past I was asked to use `#if defined()` instead of `#ifdef` and I thought it would be good to have consistent use in the same file.

Reply:
Consistency is good, and I am personally preferring `#if defined(xxx)` as then it is easy to add new conditions there if/when needed.

Reply:
Fair enough. I generally agree. If it isn't already called out maybe we should consider adding guidance to our coding standards.